src/main/java/com/plaid/client/ApiClient.java

```diff
@@ -50,7 +50,7 @@ public ApiClient() {
             public Response intercept(Chain chain) throws IOException {
                 Request originalRequest = chain.request();
                 Request requestWithUserAgent = originalRequest.newBuilder()
-                        .header("User-Agent", "Plaid Java v9.1.0")
+                        .header("User-Agent", "Plaid Java v9.2.0")
```

Comment:
ugh we should probably generate this

Reply:
This is generated

Reply:
Well I think it is, outside the scope of this release